examples/lines_bars_and_markers/fill_between_alpha.py

```diff
@@ -52,16 +49,19 @@
 #
 # Our next example computes two populations of random walkers with a
 # different mean and standard deviation of the normal distributions from
-# which the steps are drawn.  We use shared regions to plot +/- one
+# which the steps are drawn.  We use filled regions to plot +/- one
 # standard deviation of the mean position of the population.  Here the
 # alpha channel is useful, not just aesthetic.
 
+# Fixing random state for reproducibility
+np.random.seed(19680801)
+
 Nsteps, Nwalkers = 100, 250
 t = np.arange(Nsteps)
 
 # an (Nsteps x Nwalkers) array of random walk steps
-S1 = 0.002 + 0.01*np.random.randn(Nsteps, Nwalkers)
-S2 = 0.004 + 0.02*np.random.randn(Nsteps, Nwalkers)
+S1 = 0.004 + 0.02*np.random.randn(Nsteps, Nwalkers)
+S2 = 0.002 + 0.01*np.random.randn(Nsteps, Nwalkers)
```

Comment:
Note: Exchanging the two distributions results in a nicer image: S2 is drawn on top of S1. Drawing the artist with the smaller area on top looks better. One could achive the same effect by rearranging everything below, but since the data are arbitrary anyway, this change here is easiest.